packages/java-shell/package.json

```diff
@@ -11,13 +11,12 @@
     "url": "git://github.com/mongodb-js/mongosh.git"
   },
   "scripts": {
-    "browserify-shell-api": "browserify -g uglifyify src/main/js/all.js -o src/main/resources/js/all-standalone.js -dv",
+    "webpack-shell-api": "webpack --mode development",
```

Comment:
Is it intentional that we are running this in "development" and not "production" mode?

Reply:
No. :slightly_smiling_face: Fixed in 069806dcf1fb0c9d56d2d4c8c30dcf0ea79dd4ae